README.md

```diff
@@ -29,11 +29,20 @@ client. Thanks Andreas!
 
 ## Usage
 
-Reference the [Confluent.Kafka NuGet package](https://www.nuget.org/packages/Confluent.Kafka/) (version 0.11.1).
+Reference the [Confluent.Kafka](https://www.nuget.org/packages/Confluent.Kafka/) and 
+[librdkafka.redist](https://www.nuget.org/packages/librdkafka.redist/) NuGet packages (version 0.11.1).
 
-To install confluent-kafka-dotnet from within Visual Studio, search for Confluent.Kafka in the NuGet Package Manager UI, or run the following command in the Package Manager Console:
+**Important Note:** The librdkafka.redist package dependency is no longer transitive. You must now reference the 
```

Comment:
transitive? Probably correct but I would use something **I** understand, like implicit ;)

Reply:
yep. it felt funny writing it.